examples/README.rst

```diff
@@ -0,0 +1,17 @@
+Examples
+==================
```

Comment:
I think you need less `=` :)

Reply:
Ahh, yes I simply copied the markdown from README.rst I think.
I looked up the formatting and I think it is right now?
Thanks for the quick feedback!

Reply:
@bpfrh looks like not enough now :). I think you just need the same number as in  `Examples`